quickstart-node/src/services/monday.service.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiClient, ChangeColumnValueOpMutation } from '@mondaydotcomorg/api';
+import { NotFound } from 'http-errors';
+import { GetItemsInGroupQuery, GetItemsInGroupQueryVariables, ItemWithColumnValuesFragment } from 'generated/graphql';
+import { getItemsInGroup } from 'queries.graphql';
+
+const UPDATED_MONDAY_API_VERSION = '2024-10';
+
+export class MondayService {
```

Comment:
I didnt understand why we need it? shouldnt this just be functions? the constructor is empty anyway

Reply:
True, but it is for clarity. When you do `MondayService.createItem(...)` you know immediately it is related to monday. But when you do `createItem(...)` it not comes to mind immediately that it is a monday related function. I can change the function name, but I don't think this is a better solution. What do you think?